django/db/backends/oracle/base.py

```diff
@@ -242,6 +242,7 @@ def init_connection_state(self):
             self.commit()
 
     def create_cursor(self, name=None):
```

Comment:
You should add args/kwargs to the Oracle `create_cursor` as in the `BaseDatabaseWrapper.create_cursor` e.g.:
```python
def create_cursor(self, name=None, *args, **kwargs):
    return FormatStylePlaceholderCursor(self.connection)
```
Warning message will be helpful if someone pass args/kwargs.